PULL_REQUEST_TEMPLATE

```diff
@@ -5,33 +5,33 @@ Please include a summary of the change and which issue is fixed. Please also inc
 List any dependencies that are required for this change.
 -->
 
+This pull request fixes issue #<number>
+
 ## Submitter Checklist:
 
 - [ ] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
-- [ ] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
 - [ ] *Unit Tests* are updated to cover new or changed functionality
 - [ ] User-facing strings use `NSLocalizableString()`
-- [ ] New files have MPL-2.0 license header.
+- [ ] Screenshots of any new designs changes are attached.
 
 
 ## Test Plan:
-
 <!-- Any useful notes for reviewer explaining how best to test and verify. -->
 
-### Screenshots:
 
+### Screenshots:
 <!-- If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here. -->
 
 
 ## Reviewer Checklist:
 
-- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
-- [ ] Issues are assigned to at least one epic.
 - [ ] Issues include necessary QA labels:
-  - [ ] `QA/(Yes|No)`
-  - [ ] `release-notes/(include|exclude)`
-  - [ ] `bug` / `enhancement`
+  - `QA/(Yes|No)`
+  - `release-notes/(include|exclude)`
+  - `bug` / `enhancement`
 - [ ] Necessary security reviews have taken place.
```

Comment:
Wondering if this is intuitive?

Reply:
I'd say change it to `bug/enhancement/regression` and let the other two as is. Feature specific labels are added by QA team as we scan through the issues. Labels help in finding and triaging issues. Quite easy to miss issues if they are not labelled and they end in the backlog forever.

Reply:
This is about the security item, not the labels, but regarding labels:

Labels use useless unless they are used 100% consistently, and since they are unenforced they have always been unused or inconsistently used at best. If we want to use labels for categorization we need to come up with a better solution (e.g. using label prefixes and enforcing each ticket has a label with that prefix).

I think regression would be classified as `bug`.

Reply:
My bad I missed the security item below the labels. Coming back to adding labels, that's been enforced (already on desktop) but just doesn't happen on Mobile repo's as often it's supposed to happen. Any issue that QA logs always have labels. QA also go through add labels accordingly for issues logged by users who aren't collaborators on the repo.

The solution is _simple,_ if an issue is logged against a feature adding that feature label is a good practice. If its a general issue then just add bug/regression. Adding appropriate labels avoids logging duplicate issues from being logged since everyone word issue description differently.